tests/rados/test_stretch_mon_replacements.py

```diff
@@ -128,6 +129,53 @@ def run(ceph_cluster, **kw):
                 )
                 raise Exception("Mon in quorum error post removal")
 
+            if add_mon_without_location:
+                log.info(
+                    "Performing -ve scenario where the mon daemon is added without location attributes"
+                    "RFE Raised for new health warning to be generated in such cases"
+                    "ID: https://bugzilla.redhat.com/show_bug.cgi?id=2280926"
+                    "For now, just checking if the mon added without location attributes"
+                    " is not included in mon quorum"
+                )
+                flag = False
+                try:
+                    # Adding the mon to the cluster without crush locations. -ve scenario
+                    mon_obj.add_mon_service(
+                        host=mon_host,
+                        add_label=False,
+                    )
+                    flag = True
+                except Exception as err:
+                    log.info(
+                        f"Could not add mon without location attributes."
+                        f"Error message displayed : {err}"
+                    )
+                if flag:
+                    log.error(
+                        "Command execution passed for mon deployment in stretch mode without location "
+                    )
+
+                    # Checking if the new mon added is part of the quorum
+                    quorum = mon_obj.get_mon_quorum_hosts()
+                    if mon_host.hostname in quorum:
+                        log.error(
+                            f"selected host : {mon_host.hostname} mon is not present as part of Quorum post addition"
```

Comment:
Shouldn't the logger be "selected host : {mon_host.hostname} mon is present as part of Quorum post addition" ?

Reply:
Updated the comment.